google/provider.go

```diff
@@ -286,3 +287,28 @@ func linkDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	}
 	return false
 }
+
+func computeMetadataToMap(metadata []*compute.MetadataItems) map[string]string {
```

Comment:
We can name these functions flattenComputeMetadata (computeMetadataToMap) and expandComputeMetadata (mapToComputeMetadata) to match similar functions like in [`google_compute_backend_service`](https://github.com/terraform-providers/terraform-provider-google/blob/c5877743843791c4f8cdd8834e8c13594e45c349/google/resource_compute_backend_service.go#L331).